composer.json

```diff
@@ -27,7 +27,7 @@
         "ext-iconv": "*"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7.27"
+        "phpunit/phpunit": "^5.7.27 || ^8.5.39"
```

Comment:
Is there any reason not to allow 6.x or 7.x?  Or maybe there is no reason to allow them.

Reply:
I see how this can be confusing to the reader. I've now added the major versions between 5 and 8 as well. Fingers crossed that this works out well …

Reply:
Yay, it works!